templates/data-provider/index.jsx

```diff
@@ -5,31 +5,39 @@ import styles from './styles.module.css'
 
 import RegistrationLayout from 'modules/registration-layout'
 
-const DataProviderPageTemplate = () => {
-  const handleChange = () => {}
-  const handleSubmit = (event) => {
-    event.preventDefault()
-    // API call here probably
-  }
+const DataProviderPageTemplate = React.forwardRef(
+  (
+    { url, onSubmit, onUrlChange, helperMessage, variant, isFormValid = false },
+    ref
+  ) => {
+    const handleSubmit = (event) => {
+      event.preventDefault()
+      if (onSubmit) onSubmit(event)
+    }
 
-  return (
-    <RegistrationLayout>
-      <form onChange={handleChange} onSubmit={handleSubmit}>
-        <TextField
-          id="data-provider-url"
-          type="text"
-          name="dataProviderUrl"
-          label="Data provider URL"
-          helper="It can be any URL: homepage address, any data resource address, OAI-PMH endpoint or RIOXX endpoint"
-          placeholder="For example: oro.open.ac.uk"
-        />
-        <div className={styles.submitSection}>
-          <Button variant="contained" disabled>
-            Continue
-          </Button>
-        </div>
-      </form>
-    </RegistrationLayout>
-  )
-}
+    return (
+      <RegistrationLayout>
```

Comment:
What does the layout do?

Reply:
Just aligns form into middle and centre it horizontally